test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -992,18 +350,44 @@ def __init__(self, snap_args, create_tar=False):
         self.create_tar = create_tar
         self.snapshot_name = None
 
-        # Options/arguments to pbs_snapshot
-        self.diag = None
-        self.outdir = None
-        self.server_host = None
-        self.num_acct_logs = 0
-        self.num_service_logs = 0
+        # Parse the input arguments
+        # Uses Python's ternary conditional construct:
+        #    a if <condition> else b
+        self.diag = diag_path if diag_path is not None else None
```

Comment:
You are right, that was an oversight, it was needed only for a couple of the values.